source/documentation/breaking-changes/import.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Breaking Change: `@import` and global built-in functions'
+introduction: >
+  Originally, Sass used `@import` rules to load other files with a single
+  global namespace, with all built-in functions also available globally. We're
+  now deprecating both Sass `@import` rules and global built-in functions, now
+  that the module system (`@use` and `@forward` rules) has been available for
+  several years.
+---
+
+Sass historically loaded dependencies in a single global namespace using
+`@import` rules. This causes numerous problems, requiring Sass members to be
+manually namespaced to avoid conflicts, slowing down compilation when the same
+file is imported more than once, and making it very difficult for both humans
+and tools to tell where a given variable, mixin, or function comes from.
+
+The module system fixes these problems and brings Sass's modularity with the
+best practices of other modern languages, but we can't get the full benefits
+of it while `@import` remains in the language.
+
+`@import` is now deprecated as of Dart Sass 1.78.0. Additionally, we're also deprecating the global versions of Sass built-in functions that are available
+in `sass:` modules
+
+## Transition Period
+
+{% compatibility 'dart: "1.78.0"', 'libsass: false', 'ruby: false' %}{% endcompatibility %}
+
+Sass `@import` rules and global built-in function calls now emit deprecation
+warnings. While we expect Dart Sass 2.0.0 to be released soon with various
+smaller breaking changes, we don't expect to remove Sass `@import` rules or
+global built-in functions until Dart Sass 3.0.0, which will be released no
+sooner than a year after Dart Sass 1.78.0.
+
+Once Sass `@import` rules are removed from the language, all `@import` rules
+will be treated as [plain CSS `@import`s] instead. TODO: is this actually true?
```

Comment:
This will certainly _eventually_ be true. We may have an intermediate period where we make anything that used to be a Sass import throw an error before we fully move to plain CSS everywhere.